beacon-chain/p2p/types/rpc_errors.go

```diff
@@ -12,4 +12,5 @@ var (
 	ErrRateLimited            = errors.New("rate limited")
 	ErrIODeadline             = errors.New("i/o deadline exceeded")
 	ErrInvalidRequest         = errors.New("invalid range, step or count")
+	ErrBlobLTMinRequest       = errors.New("blob slot < minimum_request_epoch")
```

Comment:
I was thinknig "less than min request" which felt clear from the actual error text but this var name does look sorta weird to me. With fresh eyes I'm wondering "what is LTM and what does 'LTM in request' mean"?